src/api/user/test/e2e/user.test.js

```diff
@@ -0,0 +1,360 @@
+const request = require('supertest');
+const firebaseTesting = require('@firebase/rules-unit-testing');
+
+const { app } = require('../../index');
+const User = require('../../src/models/user');
+
+// Utility functions
+const clearData = async () => {
+  await firebaseTesting.clearFirestoreData({ projectId: 'telescope' });
+};
+
+const getUser = (id) => request(app).get(`/${id}`);
+
+const getUsers = () => request(app).get('/');
+
+const postUser1 = async () => {
+  const body = {
+    id: 10001,
+    firstName: 'Galileo',
+    lastName: 'Galilei',
+    displayName: 'Galileo Galilei',
+    isAdmin: true,
+    isFlagged: true,
+    feeds: ['https://dev.to/feed/galileogalilei'],
+    github: {
+      username: 'galileogalilei',
+      avatarUrl:
+        'https://avatars.githubusercontent.com/u/7242003?s=460&u=733c50a2f50ba297ed30f6b5921a511c2f43bfee&v=4',
+    },
+  };
+  const user = new User(body);
+  user.created = new Date().toLocaleString();
+
+  const response = await request(app).post('/').set('Content-Type', 'application/json').send(user);
+
+  return response;
+};
+
+const postUser2 = async () => {
+  const body = {
+    id: 10002,
+    firstName: 'Carl',
+    lastName: 'Sagan',
+    displayName: 'Carl Sagan',
+    isAdmin: true,
+    isFlagged: true,
+    feeds: ['https://dev.to/feed/carlsagan'],
+    github: {
+      username: 'carlsagan',
+      avatarUrl:
+        'https://avatars.githubusercontent.com/u/7242003?s=460&u=733c50a2f50ba297ed30f6b5921a511c2f43bfee&v=4',
+    },
+  };
+  const user = new User(body);
+  const response = await request(app).post('/').set('Content-Type', 'application/json').send(user);
+
+  return response;
+};
+
+// Tests
+describe('GET REQUESTS', () => {
+  beforeEach(() => clearData());
+  afterEach(() => clearData());
+
+  test('process.env.development === localhost:8088', () => {
+    expect(process.env.FIRESTORE_EMULATOR_HOST).toEqual('localhost:8088');
+  });
+
+  test('Get JSON, get all users', async () => {
+    await postUser1();
+    await postUser2();
```

Comment:
```
// Create two unique users
const galileo = await createUser({ firstName: 'Galileo', lastName: 'Galilei', /* add other properties... */ });
const carl = await createUser({ firstName: 'Carl', lastName: 'Sagan', /* add other properties... */ });
```

Now you can use `carl.user` to get the user info and `carl.response` to get the response data.